packages/json-extension/src/index.tsx

```diff
@@ -97,6 +97,7 @@ const extensions: IRenderMime.IExtension | IRenderMime.IExtension[] = [
     dataType: 'json',
     documentWidgetFactoryOptions: {
       name: 'JSON',
+      // TODO: how to translate label of the factory?
```

Comment:
That one sounds really tricky. I presume we can trick the string extraction algorithm with:

```
const trans = {__: (s: string) => s}

// ...
  name: 'JSON',
  label: trans.__('JSON'),
```
Then translate the label when the plugin is generated. But this is really ugly.